server/store/Scanning/cloudOCR.js

```diff
@@ -49,17 +48,23 @@ const transcriptParser = async paragraph => {
     words.push(word.symbols.map(symbol => symbol.text).join(''))
   );
 
-  const sentence = words.join(' ');
+  const removeCase = ['-', 'TOTAL', 'CUM', 'http', 'https'];
```

Comment:
Adding this case to reduce scanning time. 

We don't use the sentence, which contains these variables